pkgs/development/compilers/tinygo/default.nix

```diff
@@ -11,8 +11,13 @@ buildGoModule rec {
     rev = "v${version}";
     sha256 = "0x59j56y704m2hfkg78illgw9f6czrx265x887jfd989lnxphyqa";
   };
+ 
+  patches = fetchurl{
+    url = "https://raw.githubusercontent.com/c00w/patches/0437bd4a0155d2f9178b72200c4f204188a4e30e/tinygo.patch";
```

Comment:
Can you leave a comment here, how you produced this patch?
The next person should be able to re-create it.

Reply:
If you created a dummy main file. Would it be possible to do the same, when downloading the source? This way the correct `vendorSha256` should be calculated directly, no?

Reply:
Flipped this to being generated in the vendor download.